### DISCUSSION TITLE
VTK widgets: how they work with the view based on the Trame example

### DISCUSSION THREAD

**Discussion by kerim371** [Kitware/trame#602] [Q&A]
Hi,

I'm curious about the [Formula 1 picking example](https://github.com/Kitware/trame/blob/master/examples/06_vtk/Applications/SurfacePicking/app.py#L265-L283). 

Basically I can't understand the part connected with [vtk_widgets](https://github.com/Kitware/trame/blob/6bf226afa88436d94f000a4bfc8827d5d7b6f170/examples/06_vtk/Applications/SurfacePicking/app.py#L265-L283).

How to understand that?
Also I can't find source code for `trame.widgets.vtk` (i.e. `vtk_widgets`).

Thank you for sharing this amazing example!

**Comment by jourdain** [comment]
https://kitware.github.io/trame/guide/intro/widgets.html
